docs/usecase/usage-example.md

```diff
@@ -0,0 +1,87 @@
+# Use cases
+
+  This document will introduce you to the example of what Vald can do.
+  Vald is a highly scalable distributed fast approximate nearest neighbor dense vector search engine, which uses [NGT](https://github.com/yahoojapan/NGT) as the core engine of Vald, and Vald manages to integrate with Kubernetes.
+
+  In general, you cannot search your unstructured data using the inverted index, like images and videos.
+  By applying the model like BERT or VGG, you can convert your unstructured data into vectors.
+  After converting them into vectors, you can insert it to Vald and process them in Vald.
+
+  Here are some general use cases of Vald or vector search engine.
+
+## Image and video processing
+
+  ![Image and video processing](../../assets/docs/usecase_image.png)
```

Comment:
You have to confirm when we compile to `.html` file via HUGO.
And, I note about the architecture document uses `<img src="">' instead of this style.
Which style will we use in future?

Reply:
Thank you for your comment, I will confirm it via HUGO later :) 

I suggest to use `![]()` format to import the image in the future. 
The reason we used `<img src>` in architecture document was because the image was in `svg` format and `![]()` format does not support in `svg` format. 
If you support my suggestion, should I also include to change the architecture document in this PR?

Reply:
hmm...
Well, if we use `![]()` instead of `<img src=...>`, we should fix architecture document and `init.sh` in [vdaas/web](https://github.com/vdaas/web) repo.

Reply:
Oh... Should we use `<img>` tag in this PR first and fix it afterwards?